std/math.d

```diff
@@ -7774,11 +7774,11 @@ if (isFloatingPoint!(F) && isIntegral!(G))
     {
         switch (n)
         {
-        case 0:
+        case cast(G) 0:
             return 1.0;
-        case 1:
+        case cast(G) 1:
             return x;
-        case 2:
```

Comment:
Are these needed?

Reply:
indeed they don't. I have used a wrong version of DMD to test. Thanks for pointing this to me.